### DISCUSSION TITLE
menu component inside a table

### DISCUSSION THREAD

**Discussion by cfree** [twilio-labs/paste#1130] [Q&A] [answered]
is there a recommended way to use the menu component inside a table? Using the same useMenuState hook value for each instance doesn’t seem to work

![Screen Shot 2021-01-28 at 1 51 20 PM](https://user-images.githubusercontent.com/1138762/106199438-b40f4e00-6172-11eb-8fa2-02526084e920.png)

**Comment by richbachman** [comment, marked as answer]
Hi @cfree thanks for posting this question. We should probably add an example to Table docs that's showing that. We do have an example in our storybook here: https://twilio-labs.github.io/paste/?path=/story/components-table--table-layout-fixed-table

We're just repeating a single menu there, but it should also work with multiple menus.
- Menu code: https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/table/stories/index.stories.tsx#L11
- Code in table: https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/table/stories/index.stories.tsx#L587

Let us know if that helps.
